src/components/TopBar/TopBar.vue

```diff
@@ -34,18 +34,22 @@
 						:class="{'description__in-chat' : !isInCall }">
 						{{ statusMessage }}
 					</p>
-					<template v-if="conversation.description">
-						<p v-tooltip.bottom="{
-								content: renderedDescription,
-								delay: { show: 500, hide: 500 },
-								autoHide: false,
-								html: true,
-							}"
-							class="description"
-							:class="{'description__in-chat' : !isInCall }">
-							{{ conversation.description }}
-						</p>
-					</template>
+					<NcPopover v-if="conversation.description"
+						:focus-trap="false"
+						:delay="500"
+						:boundary="boundaryElement"
+						:popper-triggers="['hover']"
+						:triggers="['hover']">
+						<template #trigger="{ attrs }">
+							<p v-bind="attrs"
```

Comment:
`<p>` cannot have `aria-` attributes from `attr`, it is not valid.

But it is also not valid to make `<p>` an interactive trigger...

Reply:
it's already inside of `<a>`, which triggers settings dialog to open, and `<p>` only listening to mouseover
So, not much we can do?